src/map/options.ts

```diff
@@ -80,6 +80,8 @@ export class MapOptions extends OptionsGroup {
         mode: HTMLOption<'string'>;
         property: HTMLOption<'string'>;
     };
+    public markerOutline: HTMLOption<'boolean'>;
+    public traceLine: HTMLOption<'boolean'>;
```

Comment:
this name feels a bit too plotly specific. How about `joinPoints` instead? I'll update the code if the name works for you!